dataprofiler/data_readers/data_utils.py

```diff
@@ -415,30 +409,32 @@ def read_csv_df(
     if len(selected_columns) > 0:
         args["usecols"] = selected_columns
 
+    # if already TextIOWrapper
+    file_data = file_path
+
     # account for py3.6 requirement for pandas, can remove if >= py3.7
     is_buf_wrapped = False
-    # is_buf_open = False
+    is_file_open = False
     if isinstance(file_path, BytesIO):
```

Comment:
seems like file_path is no longer used now. Otherwise, this refactor is great!

Reply:
i think it's fixed now